From e250f9a05bb089fc35c2ec2340aa9ca00711e5b0 Mon Sep 17 00:00:00 2001
From: Thierry bordaz (tbordaz) <tbordaz@redhat.com>
Date: Mon, 21 Oct 2013 16:55:11 +0200
Subject: [PATCH 41/41] Ticket 47560: fixup memberof task does not work: task entry not added

Bug Description:

	The fixup memberof tasks is triggered by ADD of entry like:
	cn=memberOf_fixup_YYYY_MM_DD_HH_MM_SS, cn=memberOf task, cn=tasks, cn=config

	the script fixup-memberof.pl does not add this entry

Fix Description:
	Initialize the $entry variable

https://fedorahosted.org/389/ticket/47560

Reviewed by: Noriko Hosoi

Platforms tested: F17

Flag Day: no

Doc impact: no
(cherry picked from commit 05950d062b866fb7f590b19734b2eedccd1a2a2a)
---
 ldap/admin/src/scripts/fixup-memberof.pl.in |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/ldap/admin/src/scripts/fixup-memberof.pl.in b/ldap/admin/src/scripts/fixup-memberof.pl.in
index c7038f6..2355455 100644
--- a/ldap/admin/src/scripts/fixup-memberof.pl.in
+++ b/ldap/admin/src/scripts/fixup-memberof.pl.in
@@ -135,6 +135,7 @@ if ( $filter_arg ne "" )
     $filter = "filter: $filter_arg\n";
 }
 
+$entry = "${dn}${misc}${cn}${basedn}${filter}";
 $rc = DSUtil::ldapmod($entry, %info);
 
 $dn =~ s/^dn: //;
-- 
1.7.1

